package.json

```diff
@@ -1,19 +1,25 @@
 {
 	"name": "strip-debug",
-	"version": "4.0.0",
+	"version": "4.1.0",
 	"description": "Strip console, alert, and debugger statements from JavaScript code",
 	"license": "MIT",
-	"repository": "sindresorhus/strip-debug",
+	"repository": "sindresorhus/babel-plugin-strip-debug",
 	"author": {
 		"name": "Sindre Sorhus",
 		"email": "sindresorhus@gmail.com",
 		"url": "sindresorhus.com"
 	},
+	"main": "lib/index.js",
 	"engines": {
 		"node": ">=6"
 	},
 	"scripts": {
-		"test": "xo && ava"
+		"clean": "rm -rf lib",
+		"build": "babel src -d lib",
+		"test": "xo --ignore ./test/fixtures && npm run build && ava test --verbose",
+		"lint-fix": "xo --ignore ./test/fixtures --fix",
+		"test:watch": "npm run build && npm run test -- --watch",
+		"prepublishOnly": "npm run clean && npm run build"
```

Comment:
There's no point on transpiling the module code. That just adds a lot of overhead for no gain.